rmw_connext_shared_cpp/src/node_info_and_types.cpp

```diff
@@ -291,4 +293,42 @@ get_service_names_and_types_by_node(
   }
 
   return RMW_RET_OK;
-}  // extern "C"
+}
+
+rmw_ret_t
+get_service_names_and_types_by_node(
+  const char * implementation_identifier,
+  const rmw_node_t * node,
+  rcutils_allocator_t * allocator,
+  const char * node_name,
+  const char * node_namespace,
+  rmw_names_and_types_t * service_names_and_types)
+{
+  return __get_service_names_and_types_by_node(
+    implementation_identifier,
+    node,
+    allocator,
+    node_name,
+    node_namespace,
+    service_names_and_types,
+    "Request");
+}
+
+rmw_ret_t
+get_client_names_and_types_by_node(
+  const char * implementation_identifier,
+  const rmw_node_t * node,
+  rcutils_allocator_t * allocator,
+  const char * node_name,
+  const char * node_namespace,
+  rmw_names_and_types_t * service_names_and_types)
+{
+  return __get_service_names_and_types_by_node(
+    implementation_identifier,
+    node,
+    allocator,
+    node_name,
+    node_namespace,
+    service_names_and_types,
+    "Reply");
```

Comment:
I know this is the last review of the set, but I just considered that maybe the "Reply" and "Request" should be some sort of static const, so that we don't have to update them if something changes?

Reply:
Yeah, that's a good idea. What about `#define` them in [names_and_types_helpers.hpp](https://github.com/ros2/rmw_opensplice/blob/master/rmw_opensplice_cpp/src/names_and_types_helpers.hpp)?

Reply:
If you're okay with it, I can do this refactoring in a follow-up set of PRs, since this can be done in all of the rmw implementations.

Reply:
I'm good with the followup.